src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -77,18 +88,55 @@ bool AreaTrigger::CreateAreaTrigger(ObjectGuid::LowType guidlow, uint32 triggerE
 
     SetEntry(triggerEntry);
     SetDuration(spell->GetDuration());
+
+    SetUInt32Value(AREATRIGGER_DURATION, spell->GetDuration());
+
     SetObjectScale(1);
 
     SetGuidValue(AREATRIGGER_CASTER, caster->GetGUID());
     SetUInt32Value(AREATRIGGER_SPELLID, spell->Id);
     SetUInt32Value(AREATRIGGER_SPELLVISUALID, spellVisual);
     SetUInt32Value(AREATRIGGER_DURATION, spell->GetDuration());
+    SetUInt32Value(AREATRIGGER_TIME_TO_TARGET_SCALE, spell->GetDuration()); // Todo
```

Comment:
This is the time it takes for the client to play "change scale" animation, you probably should not set it to total duration